core/dispatcher/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -154,10 +154,11 @@ class ContainerPool(
             info(this, s"Shutting down: Not getting container for ${action.fullyQualifiedName} with ${auth.uuid}")
             None
         } else {
-            info(this, s"Getting container for ${action.fullyQualifiedName} with ${auth.uuid}")
-            val key = ActionContainerId(auth.uuid, action.fullyQualifiedName, action.rev)
             try {
-                getImpl(nextPosition.next(), key, () => makeWhiskContainer(action, auth)) map {
+                val myPos= nextPosition.next()
+                info(this, s"Getting container for ${action.fullyQualifiedName} with ${auth.uuid}.  myPos = $myPos  comp = ${completedPosition.cur}  slack = ${slack()}")
```

Comment:
Also, even though slack might seem expensive, it's fairly cheap given how few containers are in the data structure.